easybuild/easyconfigs/t/TensorFlow/TensorFlow-1.13.1-foss-2018b-Python-3.6.6.eb

```diff
@@ -0,0 +1,104 @@
+        'source_urls': ['https://github.com/google/protobuf/releases/download/v%(version)s/'],
+        'source_tmpl': 'protobuf-python-%(version)s.tar.gz',
+        'checksums': ['40f009cb0c190816a52fc21d45c26558ee7d63c3bd511b326bd85739b2fd99a6'],
+        'use_pip': False,
+        'start_dir': 'python',
+        'modulename': 'google.protobuf',
+    }),
+    ('absl-py', '0.7.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/a/absl-py'],
+        'checksums': ['8718189e4bd6013bf79910b9d1cb0a76aecad8ce664f78e1144980fabdd2cd23'],
+        'modulename': 'absl',
+    }),
+    ('astor', '0.7.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/a/astor'],
+        'checksums': ['95c30d87a6c2cf89aa628b87398466840f0ad8652f88eb173125a6df8533fb8d'],
+    }),
+    ('gast', '0.2.2', {
+        'source_urls': ['https://pypi.python.org/packages/source/g/gast'],
+        'checksums': ['fe939df4583692f0512161ec1c880e0a10e71e6a232da045ab8edd3756fbadf0'],
+    }),
+    ('grpcio', '1.19.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/g/grpcio'],
+        'checksums': ['2ddbca16c2e7b3f2ffc6e34c7cfa6886fb01de9f156ad3f77b72ad652d632097'],
+        'modulename': 'grpc',
+    }),
+    ('Markdown', '3.0.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/M/Markdown'],
+        'checksums': ['d02e0f9b04c500cde6637c11ad7c72671f359b87b9fe924b2383649d8841db7c'],
+    }),
+    ('tensorboard', '1.13.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/tensorboard'],
+        'source_tmpl': 'tensorboard-%(version)s-py3-none-any.whl',
+        'checksums': ['82c9c711b76949b7b3794fc319dc3d3b0fad25f7c0c5260ec4a8371b02d23da6'],
+        'unpack_sources': False,
+    }),
+    ('termcolor', '1.1.0', {
+        'source_urls': ['https://pypi.python.org/packages/source/t/termcolor'],
+        'checksums': ['1d6d69ce66211143803fbc56652b41d73b4a400a2891d7bf7a1cdf4c02de613b'],
+    }),
+    ('Werkzeug', '0.14.1', {
+        'source_urls': ['https://pypi.python.org/packages/source/W/Werkzeug'],
+        'checksums': ['c3fd7a7d41976d9f44db327260e263132466836cef6f91512889ed60ad26557c'],
+    }),
+    ('Keras-Applications', '1.0.7', {
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras-Applications'],
+        'source_tmpl': 'Keras_Applications-%(version)s.tar.gz',
+        'checksums': ['60607b2b98868983e5153bf1cc6aa468ba73adc93bc977a90edaa4bc595e69fa'],
+        'modulename': 'keras_applications',
+    }),
+    ('Keras-Preprocessing', '1.0.9', {
+        'source_urls': ['https://pypi.python.org/packages/source/K/Keras-Preprocessing'],
+        'source_tmpl': 'Keras_Preprocessing-%(version)s.tar.gz',
+        'checksums': ['5e3700117981c2db762e512ed6586638124fac5842170701628088a11aeb51ac'],
+        'modulename': 'keras_preprocessing',
+    }),
+    # FIXME new dep, should be installed from source rather using .whl?
+    ('tensorflow-estimator', '1.13.0', {
```

Comment:
@akesandgren Thoughts on this?

Reply:
good question... also bazel based i guess?

Reply:
I don't think it's worth the effort right now to try and get `tensorflow-estimator` built from source...

For now, we can take the shortcut of just installing the `.whl`, and revisit later, so I'll drop the `FIXME`.